decidim-core/lib/tasks/upgrade/decidim_deduplicate_endorsements.rake

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+namespace :decidim do
+  namespace :upgrade do
+    desc "Modify nicknames with random numbers when exists similar ones case insensitively"
```

Comment:
Copypasta, can you fix that?

Reply:
Fixed in 7f2559ffe23e3b2974f36874da9394a9ac347bb3